spec/classes/apache_spec.rb

```diff
@@ -388,6 +388,37 @@
         it { is_expected.to contain_file("/etc/httpd/conf/httpd.conf").with_content %r{^IncludeOptional "/etc/httpd/conf\.d/\*\.conf"$} }
       end
 
+      context "with Apache version < 2.4" do
+        let :params do
+          {
+            :apache_version => '2.2',
+            :rewrite_lock => '/etc/httpd/RewriteLock'
```

Comment:
shouldn't this be somewhere where httpd has write acces?
(i sure hope you're not giving it write access to its own config ;)